### ISSUE TITLE
Text Input like SDL_TEXTINPUT event

### ISSUE THREAD

**Issue by gbrls** [borodust/trivial-gamekit#57]
It would be nice to have text input like SDL's. It would be really easy to embed a repl inside the game with this.

**Comment by borodust** [comment]
True. I'll add `#'bind-any-character`. Underlying technology already has this, so it shouldn't take long to expose that in gamekit.
